aspnetcore/blazor/fundamentals/index.md

```diff
@@ -38,9 +38,33 @@ The following is an example counter component and part of an app created from a
 
 `Counter.razor`:
 
-<!-- UPDATE 8.0 Probably switch over to the BWA example when the BWA snippet sample goes up -->
+:::moniker range=">= aspnetcore-8.0"
+
+```razor
+@page "/counter"
+@attribute [RenderModeServer]
```

Comment:
I think we can do without this change. We're planning to add an option to the Blazor Web App template that will set up interactivity from the root of the app, so the Counter component won't always have the `[RenderModeServer]` attribute. The interactive render mode will get inherited from elsewhere.

Reply:
Then it seems like I should institute that change everywhere ... across all doc examples. I've been adding it to every example thus far in every topic.

Reply:
I've opened **Blazor RC2: Remove added render mode attributes from examples** #30289 to deal with this ***for RC2's release*** given that they're needed today.